src/components/FileImporter.js

```diff
@@ -103,7 +103,9 @@ class FileImporter extends Nimiq.Observable {
 
             // Make sure the data size is correct
             if (bytes.byteLength !== KeyStore.ENCRYPTED_SECRET_SIZE
-                && bytes.byteLength !== KeyStore.ENCRYPTED_SECRET_SIZE_V2) {
+                && bytes.byteLength !== KeyStore.ENCRYPTED_SECRET_SIZE_V2
+                // If an extension is present, it must consist of at least one length byte and one character byte
+                && bytes.byteLength < KeyStore.ENCRYPTED_SECRET_SIZE + 2) {
```

Comment:
suggestion: I saw there are some import/export tests, I think it would be worth adding a test for new format and old format to ensure backwards compatibility

Reply:
Added!